src/json/monaco.contribution.ts

```diff
@@ -109,6 +109,11 @@ export interface ModeConfiguration {
 	 */
 	readonly diagnostics?: boolean;
 
+	/**
+	 * Defines diagnostic codes to be ignored.
+	 */
+	readonly diagnosticCodesToIgnore?: number[];
```

Comment:
Why don't this diagnosticCodesToIgnore live under a `diagnosticsoptions` object such as the Typescript/Javascript implementation?
https://blutorange.github.io/primefaces-monaco/typedoc/interfaces/monaco.languages.typescript.diagnosticsoptions.html